sycl/test/scheduler/LinkedAlloca.cpp

```diff
@@ -0,0 +1,85 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple -I %sycl_source_dir %s -o %t.out
+// RUN: env SYCL_PRINT_EXECUTION_GRAPH=always %t.out
+// RUN: cat graph_7after_addHostAccessor.dot
+// RUN: cat graph_7after_addHostAccessor.dot | FileCheck %s
+
```

Comment:
Can we have "unit" test instead? Like this one unittests/scheduler/BlockedCommands.cpp ?

Reply:
Done.